src/main/java/com/redhat/lightblue/healthcheck/model/HealthCheckStatus.java

```diff
@@ -0,0 +1,35 @@
+package com.redhat.lightblue.healthcheck.model;
+
+public class HealthCheckStatus {
+
+    public enum Status {
+        success("{\"status\":\"success\"}"),
+        error("{\"status\":\"error\",\"message\":\"[healthcheckstatus.message]\"}");
```

Comment:
the brackets kind of make it look like it should be an array instead of a substitution key.